serial.go

```diff
@@ -91,8 +91,13 @@ type Config struct {
 	// CRLFTranslate bool
 }
 
+type ReadWriteFlushCloser interface {
+	io.ReadWriteCloser
+	Flush() error
+}
+
 // OpenPort opens a serial port with the specified configuration
-func OpenPort(c *Config) (io.ReadWriteCloser, error) {
+func OpenPort(c *Config) (ReadWriteFlushCloser, error) {
```

Comment:
This could break peoples code if they did something like

rwc, _ := serial.OpenPort(config)

rwc = something.ReturnsAReadWriteCloser()